contrib/cirrus/setup_environment.sh

```diff
@@ -118,11 +118,6 @@ fi
 case "$OS_RELEASE_ID" in
     ubuntu) ;;
     fedora)
-        # Force a crun version that has this fix: https://github.com/containers/crun/pull/819
-        # FIXME: Remove once a fixed crun made its way into Fedora
-        if test "$OS_RELEASE_VER" == "35"; then
-            yum upgrade -y https://kojipkgs.fedoraproject.org//work/tasks/684/80280684/crun-1.3-2.fc35.x86_64.rpm
```

Comment:
So, is the problem that this version has been wiped from mirrors? If so, does this actually do what you think it does? Since the CI VMs are (presumably) static, won't this just leave them with a very-old (pre-1.3.2) version of crun?

Let's see what CI does, but my gut is telling me that we should maybe, instead of deleting this block, `s/1.3-2/1.4-1/` at least until @cevich returns from PTO

Reply:
Followup: yeah, [package_versions](https://cirrus-ci.com/task/6339561314844672?logs=package_versions#L9) is showing `crun-1.3.1`, which is presumably the problem that the lines above were trying to fix. I think that, until the CI VMs get rebuilt, the solution needs to be `s/1.3-2/1.4-1/`

Reply:
Can we just do a yum update crun?

Reply:
Yes, I think that would be even better. Good thinking, and I'm sorry for not stepping back to suggest that. Juggling too many things this morning!